components/bl-autocomplete-component/README.md

```diff
@@ -1,19 +1,32 @@
 # Autocomplete
 
-Autocomplete is a component of Backendless UI-Builder designer. This allows you to select an item from a list of options.
+Autocomplete is the component of Backendless UI-Builder designer. It allows selecting one of the options from the list.
 
 <p align="center">
   <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
 </p>
 
+All options are stored in a JSON object, that can be set either by using UI-Builder settings or Codeless Logic. You can also group options by passing a list containing JSON objects to the component:
+
+<p align="center">
+  <img src="./example-images/autocomplete_component_1.png" alt="Array of objects" width="780"/>
+</p>
```

Comment:
Move it to example section. 

Also will be good to change image of JSON for real text JSON hidden under spoiler.